app/config/locale/translations/id.countries.php

```diff
@@ -165,11 +165,11 @@
     'SK' => 'Slovakia',
     'SI' => 'Slovenia',
     'SE' => 'Swedia',
-    'SZ' => 'Swaziland',
```

Comment:
Swaziland  ==> Swaziland   refer to [this](https://www.101languages.net/indonesian/country-names-indonesian/)

Reply:
I refer to https://id.wikipedia.org/wiki/Eswatini and https://en.wikipedia.org/wiki/Eswatini, should it being updated? or keep it as it is?

Reply:
> Swaziland ==> Swaziland refer to [this](https://www.101languages.net/indonesian/country-names-indonesian/)

I refer to [Wikipedia: Indonesian](https://id.wikipedia.org/wiki/Eswatini) and [Wikipedia: English](https://en.wikipedia.org/wiki/Eswatini), it is officialy renamed in 2018, also the SZ code if i refer to [ISO](https://www.iso.org/obp/ui/#iso:code:3166:SZ) it is representing Eswatini. Should it being updated? or keep it as it is?